rasa/utils/log_utils.py

```diff
@@ -30,13 +30,14 @@ def _anonymizer(
         "response",
         "slot_candidates",
         "rasa_event",
+        "rasa_events",
```

Comment:
Can't seem to leave a comment for `prediction_events` in L26, I don't think I saw it deep-copied in the place where it's being used? Please double-check 🙏🏻